### PR TITLE
[FIX] NetworkInfo network name casing

### PR DESCRIPTION
**Pull request by jpcloureiro**
**Description**

Change how a network name case is manipulated. Now it will only capitalize network names that are not `type === RPC`

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/639

### CODE REVIEW

**Review by cortisiko — APPROVED** [review]
@jpcloureiro this looks good from a QA perspective. Once the dev review is approved, feel free to merge this puppy in.

File: app/components/UI/NetworkInfo/index.tsx [hunk 1]
```diff
@@ -162,6 +162,11 @@ const NetworkInfo = (props: NetworkInfoProps) => {
     [networkProvider],
   );
 
+  const getCapitalizeNetworkName = (networkName: string) => {
+    const firstLetterUppercase = networkName.charAt(0).toUpperCase();
+    return `${firstLetterUppercase}${networkName.slice(1)}`;
+  };
```
Comment (gantunesr) [inline app/components/UI/NetworkInfo/index.tsx:168]: Can you move this to a utils file and add unit test to it?

**Review by gantunesr — COMMENTED** [review]
LGTM. Just a minor comment

**Comment by Cal-L** [comment]
Closing since included in - https://github.com/MetaMask/metamask-mobile/pull/5765

**Pull request closed without merging**